yuca/activations.py

```diff
@@ -16,7 +16,6 @@ def get_smooth_steps_fn(intervals, alpha=0.0125):
     each element contains the start and end of an interval. 
     """
     
-
```

Comment:
Check if `DoubleGaussian` and `GaussianMetaMixture` activations are used anymore. I suspect they have been made redundant (meta mixture not used, and DoubleGaussian functionality covered by the `GaussianMixture` class).